docs/conf.py

```diff
@@ -113,6 +113,7 @@
 # Exclude copy button from appearing over notebook cell numbers by using :not()
 # The default copybutton selector is `div.highlight pre`
 # https://github.com/executablebooks/sphinx-copybutton/blob/master/sphinx_copybutton/__init__.py#L82
+copybutton_exclude = ".linenos, .gp"
```

Comment:
I also noticed on that page:

> To skip all console outputs, add .go to the string above.

@stevepiercy did you intentionally not include that?  Seems also useful (prompt outputs are usually not runnable code and would interfere with copy-paste-run)

Reply:
@drammock I didn't include it because console output is not something that I would copy and paste into a terminal session. For example, in MyST source:

````
```console
A
billion
lines
of

...

stacktrace
```
````

However it might be useful in some other context of which I am not aware. Perhaps notebooks or some other utility? I selected only the options with which I am familiar. I can ammend my PR to what you think is best.